app/loader/arxiv_download.py

```
@@ -18,7 +25,9 @@
 BACKEND_ROOT = Path(__file__).resolve().parents[2]
 DEFAULT_DATA_DIR = BACKEND_ROOT / "data"
 DATA_DIR = Path(os.getenv("DATA_DIR", str(DEFAULT_DATA_DIR)))
-DATA_FILE_PATH = Path(os.getenv("ARXIV_FILE", str(DATA_DIR / "arxiv-metadata-oai-snapshot.json")))
+DATA_FILE_PATH = Path(
```

Flake8 (7.3.0)

[error] redefinition of unused 'DATA_FILE_PATH' from line 12

(F811)